controllers/storagecluster/prometheus/localcephrules.yaml

```diff
@@ -349,6 +349,32 @@ spec:
       for: 0s
       labels:
         severity: critical
+  - name: pool-usage-alert.rules
+    rules:
+    - alert: CephPoolNearFull
+      annotations:
+        description: One or more storage pools are approaching capacity limits on storage cluster {{ $labels.managedBy }}. Consider adding more storage capacity.
+        message: Storage pool is nearly full on cluster {{ $labels.managedBy }}.
+        severity_level: warning
+        storage_type: ceph
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/CephPoolNearFull.md
+      expr: |
+        ceph_health_detail{name="POOL_NEAR_FULL"} > 0
+      for: 5m
```

Comment:
Rather than waiting for 5 mins, can we reduce the triggering time to `1m`? Let me know if this not the case.